ragna/deploy/_ui/modal_configuration.py

```diff
@@ -99,6 +112,12 @@ def __init__(self, api_wrapper, **params):
         )
         self.cancel_button.on_click(self.cancel_button_callback)
 
+        self.corpus_name_input = pn.widgets.TextInput(
```

Comment:
New field required when uploading documents - also pass the name of the corpus to upload to.